samples/ImageSharp.Web.Sample/Startup.cs

```diff
@@ -37,71 +37,32 @@ public class Startup
         /// </summary>
         /// <param name="services">The collection of service desscriptors.</param>
         public void ConfigureServices(IServiceCollection services)
-        {
```

Comment:
You still need to revert all the changes bar removing `ImageSharpConfiguration`

Reply:
It makes it much more difficult for me to play around with though. We have unit tests for DI.